src/main/resources/languages/lv/messages.yml

```
@@ -35,7 +35,7 @@ guide:
     languages: 'Izvēlies savu vēlamo valodu,'
     credits: 'Slimefun4 Ieguldītāji'
   credits:
-    commit: "Ieguldījums"
+    commit: "\\\\t\nIeguldījums"
```

what is going on here...
```suggestion
    commit: "Ieguldījums"
```